lib/LocationManager.php

```diff
@@ -33,19 +45,19 @@ public static function get_locations( $caller = false ) {
 	protected static function get_locations_theme() {
 		$theme_locs = array();
 		$theme_dirs = LocationManager::get_locations_theme_dir();
-		$roots      = array(get_stylesheet_directory(), get_template_directory());
-		$roots      = array_map('realpath', $roots);
-		$roots      = array_unique($roots);
+		$roots      = array( get_stylesheet_directory(), get_template_directory() );
+		$roots      = array_map( 'realpath', $roots );
+		$roots      = array_unique( $roots );
 		foreach ( $roots as $root ) {
-			if ( !is_dir($root) ) {
+			if ( ! is_dir( $root ) ) {
 				continue;
 			}
-			$theme_locs[] = $root;
-			$root         = trailingslashit($root);
+			$theme_locs[ FilesystemLoader::MAIN_NAMESPACE ][] = $root;
```

Comment:
This might be undesirable if a user is using a custom loader (via a filter) and have customised the main namespace name. Should a define a constant on the `Loader ` class within Timber rather than use the one from the Twig Filesystem Loader?

Reply:
Addressed in https://github.com/timber/timber/pull/1791/commits/f31adba1691712e44102f46611a2b523b2ae04c8